### PR TITLE
Prepare Role resource grouping with Access

### PR DESCRIPTION
**Pull request by rhybrillou**
## Description

As part of the effort to simplify the permission management in the product, the `Role` resource will be grouped with the `Access` one.
This change announces the grouping and flags the code portions to alter in order to make this change happen.

## Checklist
- [ ] Investigated and inspected CI test results
~~- [ ] Unit test and regression tests added~~
- [ ] Evaluated and added CHANGELOG entry if required
~~- [ ] Determined and documented upgrade steps~~
~~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

If any of these don't apply, please comment below.

## Testing Performed

Mostly comment addition in the code.
CI should be enough.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Deprecated Features
 - ROX-12620: We continue to simplify access control management by grouping some permissions in permission sets. As a result:
   - The permission `WorkflowAdministration` will deprecate the permissions `Policy, VulnerabilityReports`.
+- ROX-14398: We continue to simplify access control management by grouping some permissions in permission sets. As a result:
+  - The permission `Access` will deprecate the permissions `Role`.
+  - The default role `Scope Manager` will be removed.
```
Comment (dhaus67) [inline CHANGELOG.md:23]: We've added the `required actions` section below to address some feedback regarding customers not necessarily knowing what they have to do to remediate the deprecation. We should cover the role deprecation within there as well.

Especially with the removal of the `Scope Manager` role, this _does_ require an action if its used as a default minimum access role / group within auth providers.

**Review by dhaus67 — COMMENTED** [review]
Just one comment regarding the changelog and what it entails.

Could you also include the role removal / migrations to [ROX-14398](https://issues.redhat.com/browse/ROX-14398)? Think its worth to explicitly call it out besides the role resource migration.

File: CHANGELOG.md (hunk 1 above)
Comment (md2119) [inline CHANGELOG.md:23]: when will it be removed?

**Review by md2119 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rhybrillou) [reply CHANGELOG.md:23]: Target is 3.76 if this PR is merged before 3.74 release start, 3.77 otherwise.

**Review by dhaus67 — APPROVED** [review]

**Pull request merged**